compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -10018,6 +10024,13 @@ private STNode parseClientDeclOrVarDecl(STNode metadata, STNode publicQualifier,
         }
     }
 
+    private void invalidateDocumentation(STMetadataNode metadataNode) {
+        STNode documentationString = metadataNode.documentationString;
+        if (documentationString != null) {
+            addInvalidNodeToNextToken(documentationString, DiagnosticErrorCode.ERROR_INVALID_DOCUMENTATION);
+        }
```

Comment:
Here if the `metadataNode.annotations` are not null we should attach invalid node to annotations. otherwise we can do `addInvalidNodeToNextToken`

Reply:
Updated in [c15a567](https://github.com/ballerina-platform/ballerina-lang/pull/37357/commits/c15a5671e628d9de63c0c80bbd07ea96c6036b6b).

Reply:
Fixed attaching to the client keyword in d7a9d01.